packages/nc-gui/components/account/Token.vue

```diff
@@ -28,6 +37,18 @@ const pagination = reactive({
   total: 0,
   pageSize: 10,
 })
+
+// hide or show token
+const showToken = (tokenId: string) => {
```

Comment:
better name as `hideOrShowToken` since you will hide the token in this function. By renaming it, that comment at L41 is not required.